src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +40,90 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ??? // here is my greatest solution
+def not(b: Boolean): Boolean =
+  if b then false
+  else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+def and(left: Boolean, right: Boolean): Boolean =
```

Comment:
Suppose `left` is true. What value fully determine the result value of conjunction?